@jiayuehua, Thanks for your PR, this is ci error log, please take a look:

````
CMake Error at D:/installed/x64-windows/share/vcpkg-cmake-config/vcpkg_cmake_config_fixup.cmake:36 (message):
  'D:/packages/openblas_x64-windows/debug/share/cmake/OpenBLAS' does not
  exist.
````

--- a/ports/openblas/portfile.cmake
+++ b/ports/openblas/portfile.cmake
@@ -68,7 +68,7 @@ vcpkg_cmake_configure(
 vcpkg_cmake_install()
 vcpkg_copy_pdbs()
 
-vcpkg_cmake_config_fixup(CONFIG_PATH share/cmake/OpenBLAS)
+vcpkg_cmake_config_fixup(CONFIG_PATH lib/cmake/OpenBLAS)
 
 if (EXISTS "${CURRENT_PACKAGES_DIR}/bin/getarch${VCPKG_HOST_EXECUTABLE_SUFFIX}")
     vcpkg_copy_tools(TOOL_NAMES getarch AUTO_CLEAN)

--- a/ports/openblas/portfile.cmake
+++ b/ports/openblas/portfile.cmake
@@ -1,7 +1,7 @@
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO xianyi/OpenBLAS
-    REF v${VERSION}
+    REF "v${VERSION}"
     SHA512 fe66e3a258ca1720764ed243f6d61017d6ef14bd33b76f20b19b34754096ec2be9fbeb1a78743f38ee71381746d6af9a1c16a8f3982e423afec422fcb50852d0
     HEAD_REF develop
     PATCHES

--- a/versions/o-/openblas.json
+++ b/versions/o-/openblas.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "488e2ec60894835ebacaa8b2399ce3591b1313cf",
+      "git-tree": "c876665c0fa5b8d427ee9cadd4185a1a08d008da",
       "version": "0.3.24",
       "port-version": 0
     },

